features/admins/admin_post_faqs.feature

```diff
@@ -2,85 +2,88 @@
-
-  Scenario: Edit FAQ
-    Given I have posted a FAQ
-    When I follow "Admin Posts"
-      And I follow "Archive FAQ" within "#header"
-      And I follow "Edit"
-      And I fill in "Answer*" with "Number 1 posted FAQ, this is, and Yoda approves."
-      And I press "Post"
-    Then I should see "ArchiveFaq was successfully updated"
-      And I should see "Yoda approves"
-    When I go to the archive_faqs page
-      And I follow "Edit"
+    When I follow "Edit"
       And I fill in "Answer*" with "New Content, yay"
       And I press "Post"
+    Then I should see "New Content, yay"
+      And I should not see "Some text"
 
-  Scenario: Post a FAQ that is a Translation of another
+  Scenario: Post a translated FAQ for a locale, then change the locale's code.
     Given basic languages
+      And I am logged in as an admin
+
+    # Post "en" FAQ
     When I go to the archive_faqs page
-    Then I should see "Some commonly asked questions about the Archive are answered here"
-      And I should not see "Some text"
-    When I am logged in as an admin
-    When I follow "Admin Posts"
-      And I follow "Archive FAQ" within "#header"
-      And I should not see "Some text"
-    When I follow "New FAQ Category"
+      And I follow "New FAQ Category"
       And I fill in "Question*" with "What is AO3?"
       And I fill in "Answer*" with "Some text, that is sufficiently long to pass validation."
       And I fill in "Category name*" with "New subsection"
       And I fill in "Anchor name*" with "whatisao3"
-    When I press "Post"
+      And I press "Post"
     Then I should see "ArchiveFaq was successfully created"
-      And 0 emails should be delivered
 
-    
+    # Translate FAQ to "de"
+    When I follow "Archive FAQ"
+      And I select "Deutsch" from "language_id"
+      And I press "Go" within "div#inner.wrapper"
+      And I follow "Edit"
+      And I fill in "Question*" with "Was ist AO3?"
+      And I fill in "Answer*" with "Einige Text, das ist lang genug, um die Überprüfung bestanden."
+      And I fill in "Category name*" with "Neuer Abschnitt"
+      And I check "Question translated"
+      And I press "Post"
+    Then I should see "ArchiveFaq was successfully updated."
+      And I should not see "New subsection"
+      And I should see "Neuer Abschnitt"
+      And I should see "Was ist AO3?"
+      And I should see "Einige Text"
+
+    # Change locale "de" to "ger"
+    When I go to the locales page
+      And I follow "Edit"
+    Then I should see "Deutsch" in the "Name" input
+    When I fill in "locale_iso" with "ger"
+      And I press "Update Locale"
+    Then I should see "Your locale was successfully updated."
+      And I should see "Deutsch ger"
+
+    # The current session preference remains "de", the category name falls back to the default locale,
+    # but no "de" questions will be found.
+    When I go to the archive_faqs page
+      And I follow "New subsection"
```

Comment:
The test dies here without `config.i18n.fallbacks = true`. The FAQ title will be `nil` for locale "ger", which means:

- This link to the FAQ (on the FAQ index) is bare, appearing as `/faq/new-subsection?language_id=de`.
- Following that link (or "Show") will crash the request when adding the title to the page subtitle:
  https://github.com/otwcode/otwarchive/blob/6b99402ac54eba0b220e85163408c777322c2251/app/controllers/archive_faqs_controller.rb#L34